src/lib/containers/oauth/OAuthManagement.tsx

```diff
@@ -1,16 +1,32 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { Table } from 'react-bootstrap'
 import { formatDate } from '../../utils/functions/DateFormatter'
 import moment from 'moment'
 import { Button } from 'react-bootstrap'
 import { useGetOAuthClientInfinite } from '../../utils/hooks/SynapseAPI'
+import { CreateOAuthModal } from './CreateOAuthClient'
+import { OAuthClient } from '../../utils/synapseTypes/OAuthClient'
 
 export const OAuthManagement: React.FunctionComponent = () => {
+  const [isShowingCreateClientModal, setIsShowingCreateClientModal] =
+    useState(false)
+  const [isEdit, setIsEdit] = useState<boolean>(false)
+  const [selectedClient, setSelectedClient] = useState<OAuthClient>()
+  const [isShowingConfirmModal, setIsShowingConfirmModal] = useState(false)
```

Comment:
Why control the confirm modal's state here instead of in the CreateOAuthClientModal?

Reply:
This was from an error I got while running tests. When testing for OAuthManagement I constantly got a type error of `useMutateOAuthClient` from `CreateOAuthClient.tsx` not being a function. I ended up having the component not render unless either one of the modal's render which made me put the confirm modal's state in OAuthManagement. It was a hacky way of solving the problem and am open to different solutions